components/tools/OmeroJava/test/integration/LightAdminRolesTest.java

```diff
@@ -1276,6 +1384,9 @@ public void testChownAllBelongingToUser(boolean isPrivileged, String groupPermis
         loginUser(lightAdmin);
         client.getImplicitContext().put("omero.group", "-1");
         /* transfer can proceed only if chownPassing boolean is true */
+        /* Check on one selected object only (sentProj1OtherGroup) the value
```

Comment:
might have to make a special note about variable names in comments as refactoring to change name may not catch them